packages/dht/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "@streamr/dht",
+  "version": "0.0.1",
+  "description": "Streamr Network DHT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/streamr-dev/network-monorepo.git",
+    "directory": "packages/dht"
+  },
+  "license": "STREAMR NETWORK OPEN SOURCE LICENSE",
+  "author": "Streamr Network AG <contact@streamr.network>",
+  "scripts": {
+    "sandbox": "ts-node src/index.ts",
+    "prebuild": "./proto.sh",
+    "build": "tsc -b tsconfig.node.json",
+    "build-browser": "webpack --mode=development --progress",
+    "check": "tsc -p ./tsconfig.json --noEmit",
+    "clean": "jest --clearCache || true; rm -rf dist *.tsbuildinfo node_modules/.cache src/proto || true",
+    "coverage": "jest --coverage",
+    "eslint": "eslint --cache --cache-location=node_modules/.cache/.eslintcache/ '*/**/*.{js,ts}'",
+    "prepare": "npm run build",
+    "prepublishOnly": "npm run clean && NODE_ENV=production tsc -b tsconfig.node.json",
+    "test": "jest",
+    "test-browser": "npm run build-browser && karma start karma.config.js",
+    "test-unit": "jest test/unit",
+    "test-integration": "jest test/integration",
+    "test-end-to-end": "jest test/end-to-end",
+    "test-correctness": "jest test/correctness",
```

Comment:
Is there a need that the `correctness` tests are a separate test group? Most tests are about correctness, so maybe could rename this if it needs to be a separate entity.